src/riak_dt.proto

```diff
@@ -154,6 +156,18 @@ message SetOp {
     repeated bytes removes = 2;
 }
 
+
+/*
+ * An operation to update a GSet, on its own.
+ * GSet members are opaque binary values, you can only add
+ * them to a Set.
+ */
+message GSetOp {
```

Comment:
`SetOp` can be re-used. If `removes` are part of a `SetOp` for `GSET`, Riak should return an error.